recipes/librdkafka/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+from conans import CMake, ConanFile, tools
+
+
+class LibrdkafkaConan(ConanFile):
+    name = "librdkafka"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/edenhill/librdkafka"
+    description = "Librdkafka is an Apache Kafka C/C++ library designed with message delivery reliability and high performance in mind."
+
+    topics = ("kafka", "librdkafka")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "zlib": [True, False],
+        "zstd": [True, False],
+        "plugins": [True, False],
+        "ssl": [True, False],
+        "sasl": [True, False],
+        "lz4": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "zlib": False,
+        "zstd": False,
+        "plugins": False,
+        "ssl": False,
+        "sasl": False,
+        "lz4": False,
+    }
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    sources_folder = "sources"
+    _cmake = None
```

Comment:
```suggestion
    _cmake = None

    @property
    def _source_subfolder(self):
        return "source_subfolder"
```

Any custom method or attribute must be protected.